packages/runtime/src/next/index.tsx

```diff
@@ -1,6 +1,11 @@
 export type { PageProps } from './components/page'
 export { Page } from './components/page'
-export type { MakeswiftPage, MakeswiftPageDocument, MakeswiftPageSnapshot, Sitemap } from './client'
```

Comment:
Seems like we're making a breaking _type_ change here. Maybe we should call that out in the changeset, upgrade guide, etc.?

Reply:
Resolved, called out in the changeset